services/movie.py

```diff
@@ -28,13 +36,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Avoid using mutable default arguments like `list` for `genres_ids` and `actors_ids`. Use `None` as the default value and initialize with an empty list inside the function if needed.